prow/external-plugins/needs-rebase/plugin/plugin.go

```diff
@@ -32,8 +32,8 @@ import (
 )
 
 const (
-	pluginName         = "needs-rebase"
-	needsRebaseLabel   = "needs-rebase"
+	PluginName         = "needs-rebase"
```

Comment:
Personally I would still like to see this either not produce a lint error (because we increased the minimum confidence) or write a command to satisfy the linter.